Feature/#11 add api decibel

yohan-kang opened the pull request:
## Summary

## Tasks
noise add api 추가 
postman 테스트 완료

minjo-on committed:
refactor: review domain 삭제

minjo-on committed:
refactor: user domain 분리

minjo-on committed:
refactor: controller 경로 수정

rmvmf96 committed:
NoiseCreateResponse Dto add

rmvmf96 committed:
NoiseCreateRequest 수정
user_id 추가

rmvmf96 committed:
NoiseService 로직 생성 , api test 완료

minjo-on commented on src/main/java/com/soridam/server/noise/controller/NoiseApiController.java:
```diff
@@ -9,4 +15,13 @@
+        Long noiseId = noiseService.createNoise(noiseCreateRequest);
+        NoiseCreateResponse response = new NoiseCreateResponse(200, "success", noiseId);
+        return ResponseEntity.ok(response);
+    }
```
응답 객체 따로 만드는 방법 말구 ResponseEntity.status(CREATED); 이렇게 응답하는게 좋을 것 같습니다 !

minjo-on commented on src/main/java/com/soridam/server/noise/service/NoiseService.java:
```diff
@@ -1,15 +1,55 @@
+		// Noise 저장
+		Noise savedNoise = noiseRepository.save(noise);
+		return savedNoise.getId();
+	}
```
Noise의 create method를 사용해주시고, user custom exception 구현하여 적용해 주세요!

minjo-on commented on src/main/java/com/soridam/server/user/repository/JpaUserRepository.java:
```diff
@@ -0,0 +1,11 @@
+
+public interface JpaUserRepository extends JpaRepository<User, Long> {
+    Optional<User> findById(Long id);
+}
```
jpa에서 선언 안해도 지원해주기 때문에 지우셔도 될 것 같습니다!

rmvmf96 committed:
Merge branch 'develop' of https://github.com/9oormthon2024-T5-SoriSoop/soridam-server into feature/#11-add-api-decibel

# Conflicts:
#	src/main/java/com/soridam/server/noise/controller/NoiseApiController.java
#	src/main/java/com/soridam/server/noise/service/NoiseService.java
#	src/main/java/com/soridam/server/user/controller/UserApiController.java
#	src/main/java/com/soridam/server/user/domain/User.java
#	src/main/java/com/soridam/server/user/exception/UserExceptionCode.java
#	src/main/java/com/soridam/server/user/repository/JpaUserRepository.java
#	src/main/java/com/soridam/server/user/service/UserService.java

minjo-on committed:
feat: noise 상세 조회 구현

rmvmf96 committed:
Merge branch 'feature/#17-user-mypage-noise-list-detail' of https://github.com/9oormthon2024-T5-SoriSoop/soridam-server into feature/#11-add-api-decibel

# Conflicts:
#	src/main/java/com/soridam/server/noise/service/NoiseService.java

rmvmf96 committed:
Noise 서비스 계층 Noise reate method 사용하는 로직으로 수정
서비스 로직 user 부분 user custom exception 적용
Noise 생성 컨트롤러 ResponseEntity.status(CREATED) 형식으로 수정

rmvmf96 committed:
UserRepsitory 계층 findById 삭제
NoiseService  예외처리 UserNotFoundException 사용

minjo-on committed:
Merge branch 'develop' into feature/#11-add-api-decibel
